tests/ClassicalField/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_executable(Laplace Laplace.f90)
+set_target_properties(Laplace PROPERTIES LINKER_LANGUAGE Fortran)
```

Comment:
Is the linker language not getting picked up from the .f90 in the source file name?  I would have hoped that this wouldn't haven been required to set explicitly.